### DISCUSSION TITLE
Does OQS Provide Support for Blockchain Applications?

### DISCUSSION THREAD

**Discussion by galaxyonymous** [open-quantum-safe/liboqs#1932] [Q&A] [answered]
I am working on my project related to Post Quantum Blockchain Crypto Currencies, and I find Liboqs very useful, but I am finding a way to integrate OQS libraries in my smart contracts of solidity for my Project, is there anyway out or did someone already give it a try?

Need suggestion as well that is this really worth it using it in Smart Contract applications or not?

**Comment by SWilson4** [comment, marked as answer]
We don't provide direct support for blockchain applications _per se_, but if you have general questions about the library and its use we can probably help.

**Reply by galaxyonymous** [reply]
Thank you very much for Clarification,
SWilson.
